examples/flash_example.rs

```diff
@@ -48,8 +48,13 @@ impl FlashBlock {
         let addr = self.addr() - 0x10000000;
         defmt::assert!(addr & 0xfff == 0);
 
+        let flash_iap = flash::FlashIAP::new(true);
+
         cortex_m::interrupt::free(|_cs| {
-            flash::flash_range_erase_and_program(addr, data, true);
+            // SAFETY: we made sure the requirements are met
+            unsafe {
```

Comment:
Interesting, nightly does not warn for that 🤔

Reply:
Probably because the long-term goal is to `#[deny(unsafe_op_in_unsafe_fn)]`, which means that the explicit `unsafe` block becomes necessary.

So the right fix might be to add `#![deny(unsafe_op_in_unsafe_fn)]` at the top of the file, and keep the `unsafe` block.